operations/admin_all_configs.go

```diff
@@ -63,17 +70,19 @@ func fetchAllProjectConfigs() cli.Command {
 
 // fetchAndWriteConfig downloads the most recent config for a project
 // and writes it to "project_name.yml" locally.
-func fetchAndWriteConfig(c *legacyClient, project string) error {
-	grip.Infof("Downloading configuration for %s", project)
-	versions, err := c.GetRecentVersions(project)
+func fetchAndWriteConfig(ac *legacyClient, rc *legacyClient, project string, validate bool) error {
```

Comment:
does it make sense to seperate these options entirely, leaving the fetchAndWriteConfig function as it is, and then just having an option to the validator to just validate all files in a directory? The code would be simpler, I think.

also I'm not sure what the local validator does vs the remote one or other ways of calling into the validator, but some context might be useful.

Reply:
yeah! if you could. 

We've been talking about building better admin tools for managing projects, and I think keeping operations seperate will make that easier.

Reply:
Made it so that the validate command can take a directory and will validate everything in it